src/tableaus/low_order_rk_tableaus.jl

```diff
@@ -1884,3 +1884,175 @@ function SIR54ConstantCache(T::Type{<:CompiledFloats}, T2::Type{<:CompiledFloats
                        btilde1, btilde2, btilde3, btilde4, btilde5, btilde6, btilde7, c2,
                        c3, c4, c5, c6, c7)
 end
+
+struct Alshina2ConstantCache{T, T2} <: OrdinaryDiffEqConstantCache
+    a21::T
+
+    b1::T
+    b2::T
+
+    c2::T2
+    
+end
+
+function Alshina2ConstantCache(T::Type{<:Float64}, T2::Type{<:Float64})
+    a21 = convert(T, 0.6666666666666666)
+
+    b1 = convert(T, 0.25)
+    b2 = convert(T, 0.75)
+
+    c2 = convert(T2, 0.6666666666666666)
+    
+    Alshina2ConstantCache(a21, b1, b2, c2)
+end
+
+struct Alshina3ConstantCache{T, T2} <: OrdinaryDiffEqConstantCache
+    a21::T
+    a31::T
+    a32::T
+
+    b1::T
+    b2::T
+    b3::T
+
+    c2::T2
+    c3::T2
+    
+end
+
+function Alshina3ConstantCache(T::Type{<:Float64}, T2::Type{<:Float64})
+    a21 = convert(T, 0.5)
+    a31 = convert(T, 0.0)
+    a32 = convert(T, 0.75)
+
+    b1 = convert(T, 0.2222222222222222)
+    b2 = convert(T, 0.3333333333333333)
+    b3 = convert(T, 0.4444444444444444)
+
+    c2 = convert(T2, 0.5)
+    c3 = convert(T2, 0.75)
+    
+    Alshina3ConstantCache(a21, a31, a32, b1, b2, b3, c2, c3)
+end
+
+struct Alshina4ConstantCache{T, T2} <: OrdinaryDiffEqConstantCache
+    a21::T
+    a31::T
+    a32::T
+    a41::T
+    a42::T
+    a43::T
+    
+
+    b1::T
+    b2::T
+    b3::T
+    b4::T
+
+    c2::T2
+    c3::T2
+    c4::T2
+    
+end
+
+function Alshina4ConstantCache(T::Type{<:Float64}, T2::Type{<:Float64})
+    a21 = convert(T, 0.5)
+    a31 = convert(T, 0.0)
+    a32 = convert(T, 0.5)
+    a41 = convert(T, 0.0)
+    a42 = convert(T, 0.0)
+    a43 = convert(T, 1.0)
+
+    b1 = convert(T, 0.16666666666666666)
+    b2 = convert(T, 0.3333333333333333)
+    b3 = convert(T, 0.3333333333333333)
+    b4 = convert(T, 0.16666666666666666)
+
+    c2 = convert(T2, 0.5)
+    c3 = convert(T2, 0.5)
+    c4 = convert(T2, 1.0)
+    
+    Alshina4ConstantCache(a21, a31, a32, a41, a42, a43, b1, b2, b3, b4, c2, c3, c4)
+end
```

Comment:
This is just RK4?

Reply:
Do I need to resolve anything here? This is indeed just RK4.

Reply:
Delete it because it's not useful.